File: Sem03/DNP1/Introduction/S01_Ex03/Person.cs
using System;

namespace Ex3
{
    class Person
    {
        public string Name { get; set; }

        public Person(string name)
        {
            this.Name = name;
        }

        public void Introduce()
        {
            Console.Write($"Hi, my name is: {Name} ");
        }
    }
}

File: Sem01/SDJ1/WORKSHOP_2/src/Person.java
public class Person
{
  private String name;
  private Pet pet;

  public Person(String name)
  {
    this.name = name;
  }
  public String getName()
  {
    return name;
  }
  public Pet getPet()
  {
    return pet;
  }
  public String getPetsName()
  {
    return pet.getName();
  }
  @Override public String toString()
  {
    return "Person{" + "name='" + name + '\'' + ", pet=" + pet + '}';
  }
}
